src/Microsoft.Diagnostics.Runtime/src/Windows/Minidump.cs

```diff
@@ -290,6 +284,15 @@ private async Task<ThreadReadResult> ReadThreadData(Stream stream)
                 Threads = threadBuilder.ToImmutable()
             };
         }
+
+        private static async Task<int> ReadAsync(Stream stream, byte[] buffer, int count)
+        {
+#if NETCOREAPP3_1 || NET5_0
```

Comment:
```suggestion
#if NETCOREAPP
```

https://github.com/dotnet/designs/blob/main/accepted/2020/net5/net5.md#preprocessor-symbols